docs/src/calibration.md

```diff
@@ -199,33 +201,160 @@ You would start the job with a command such as `qsub name_of_job_script` for PBS
-- `caldir`: you can change the name and path of your calibration output directory
-- `training_locations`: the default is all locations on land, but you can change this to a subset of land coordinates
-
-You should also modify the files:
-- `priors.jl`, to give your parameters and their distribution
-- `forward_mode.jl` or `forward_model_bucket.jl`, to ensure it reads the parameters from `priors.jl` and uses them
-
-other things to consider:
-- `observationseries_era5.jl`: currently the target data is era5. This could be changed to another dataset, or a perfect model target
-- `observationseries_era5.jl`: the noise is currently set to 5^2 (flat noise, in W m^-2), this should be changed to desired noise
-- you could change the EnsembleKalmanProcesses settings, set in `EKP.EnsembleKalmanProcesses()` in `calibrate_land.jl`
-- you can adjust the ClimaCalibrate backend, see the [documentation](https://clima.github.io/ClimaCalibrate.jl/dev/backends/)
-- Because the variable names are currently different in era5 file, you may have to add variables to map in observationseries_era5.jl (for example, "lhf" => "mslhf")
-
-Also, note that currently:
-- the temporal resolution of observation_maps are seasonal (3 months average)
-- the length of calibration is 1 year (data used after spinup)
-- the entire globe is used
-These could also be changed in the code, but would currently requires significant changes.
-
-Finally, note that the HPC job script and command will slightly differ between slurm (for example central or clima) and pbs (for example derecho).
+## Configure your land calibration
+
+For configuring the land calibration, you can configure the optimization method
+used by EnsembleKalmanProcesses.jl, the observations and how they are
+preprocessed, and the simulation itself. For most settings, you can modify
+the `CalibrateConfig` struct. See the example below.
+
+```julia
+CalibrateConfig(;
+    short_names = ["swu"],
+    minibatch_size = 2,
+    n_iterations = 10,
+    sample_date_ranges = [("2007-12-1", "2008-9-1"),
+                          ("2008-12-1", "2009-9-1"),
+                          ("2009-12-1", "2010-9-1"),
+                          ("2010-12-1", "2011-9-1")],
+    extend = Dates.Month(3),
+    spinup = Dates.Month(3),
+    nelements = (101, 15),
+    output_dir = "experiments/calibration/land_model",
+    rng_seed = 42,
+)
+```
+
+With the configuration above, a calibration is being done using the `swu`
+observation. The calibration will run for 10 iterations, and each iteration will
+use a minibatch size of 2. The start and end dates of the simulation is
+automatically determined by `sample_date_ranges`, `spinup`, and `extend`. The
+amount to spin up the simulation for is three months and the amount to run the
+simulation for past the dates in `sample_date_ranges` is also three months.
+Since the minibatch size is 2, the first iteration of the calibration will run
```

Comment:
could you clarify this a bit more? Minibatch = N is in units of years? dos this relate to the number of elements in sample_date_ranges at all?

Reply:
oh, i see this below